src/controllers/metadata.js

```diff
@@ -1,21 +1,22 @@
 'use strict'
 const config = require('../services/config')
 
+const base = config.getIn(['server', 'base_uri'])
 const metadata = {
   currency_code: config.getIn(['currency', 'code']),
   currency_symbol: config.getIn(['currency', 'symbol']),
   condition_sign_public_key: config.getIn(['keys', 'ed25519', 'public']),
   notification_sign_public_key: config.getIn(['keys', 'notification_sign', 'public']),
   urls: {
-    health: '/health',
-    transfer: '/transfers/:id',
-    transfer_fulfillment: '/transfers/:id/fulfillment',
-    transfer_state: '/transfers/:id/state',
-    connectors: '/connectors',
-    accounts: '/accounts',
-    account: '/accounts/:name',
-    subscription: '/subscriptions/:id',
-    subscription_notification: '/subscriptions/:subscription_id/notifications/:notification_id'
+    health: base + '/health',
```

Comment:
Yes, but is there any advantage to using absolute URLs here? In general I think relative URLs are more robust because for example the same server can often be accessed from multiple hostnames (`localhost` and the DNS name is one common example). Even though it might work this way, the burden of proof should probably be on justifying using absolute URLs due to these considerations.